src/context.rs

```diff
@@ -63,28 +67,49 @@ impl Context {
         }
     }
 
+    /// Increases the reference count on self by one. This prevents self from being destroyed
+    /// until a matching call to drop() is made.
+    /// 
+    /// The number of references to a cairo_t can be get using Context::get_reference_count().
     pub fn reference(&self) -> Context {
         unsafe {
             Context(ffi::cairo_reference(self.get_ptr()))
         }
     }
 
-    pub fn ensure_status(&self){
+    pub fn ensure_status(&self) {
         self.status().ensure_valid();
     }
 
+    /// Creates a new Context with all graphics state parameters set to default values
```

Comment:
It's just what everybody does. ;) I know it would be frustrating to do by hand. Might be possible to do with some regex incantations.